package-lock.json

```diff
@@ -8226,20 +8226,20 @@
       "dependencies": {
         "abbrev": {
           "version": "1.1.1",
-          "resolved": "",
+          "resolved": false,
```

Comment:
Running this branch locally, with `npm i`, the resolved values throughout remain unchanged as `"resolved": "",`

Reply:
I can rerun the npm install and recreate the package-lock file after the #511 is merged in

Reply:
I nuked the package.lock file and reran the installation. Should be fine now